src/app/introduction/activity-3/activity-3.component.ts

```diff
@@ -1,12 +1,11 @@
-import { Component, OnInit, Output, EventEmitter, ViewChild, Renderer, ViewContainerRef } from '@angular/core';
-import { Observable } from 'rxjs/Rx';
-import { Http, Headers, Response, RequestOptions } from '@angular/http';
-import { DashboardService } from '../../services/dashboard.service';
+import {Component, OnInit, Renderer2, ViewChild, ViewContainerRef} from '@angular/core';
+
+import {HttpClient} from '@angular/common/http';
+import {DashboardService} from '../../services/dashboard.service';
 import 'webrtc-adapter';
-import { webcamEnum } from './webcamEnum';
-import { SharedDataService } from '../../services/shared.data.service';
-import { ToastsManager } from 'ng2-toastr/ng2-toastr';
-import { LanguageService } from '../../services/language.service';
+import {webcamEnum} from './webcamEnum';
+import {SharedDataService} from '../../services/shared.data.service';
+import {LanguageService} from '../../services/language.service';
```

Comment:
Spacing while importing components is not fine throughout this PR.  Please correct it.

Reply:
Spacing issue again